lib/plugins/features/movies/index.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+exports.register = (server, options, next) => {
+
+  next();
+
+};
+exports.register = (server, options, next) => {
+
+  server.route([{
+    method: 'POST',
+    path: '/movies',
+    config: {
+      handler: (request, reply) => {
+        reply('hello world');
+      }
+    }
+  }]);
+
+  next();
+
+};
+
+exports.register.attributes = {
+  name: 'movies'
+};
+
```

Comment:
I think this is an extraneous newline

Reply:
Done.

Reply:
Don't we usually add newlines at the end of each file?

Reply:
Yes but there were two